contributing/ci-consortium.txt

```diff
@@ -93,33 +89,15 @@ https://github.com/openmicroscopy/Imperial-FLIMfit
 
 		This job is used to build the FLIMfit downloads page
 
-		#. Checks out the `dev_5_0` branch of
+		#. Checks out the `develop` branch of
 		   https://github.com/openmicroscopy/ome-release
-		#. Merge PRs opened against `dev_5_0`
+		#. Merge PRs opened against `develop`
 		#. Copy the FLIMfit Windows artifacts over SSH to
 		   :file:`/ome/apache_repo/public/flimfit/RELEASE`
 		#. Build the FLIMfit downloads pages using :command:`make flimfit`
 		#. Copy the FLIMfit downloads page over SSH to
 		   :file:`/ome/apache_repo/public/flimfit/RELEASE`
 
-OMERO.figure
```

Comment:
Because it isn't a consortium job any more and this page is all about partner projects not mainline things.

We can put it on the release jobs page instead if you like, I've added the figure-help builds to the docs builds page already.